.github/workflows/CI.yml

```
@@ -42,28 +42,30 @@ jobs:
       - name: Prettier Format
         uses: creyD/prettier_action@v4.3
         env:
-            GITHUB_TOKEN: ${{ secrets.TOKEN }} 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          prettier_options: '--write "**/*.{js,ts,tsx,json,md,css}"'
+          prettier_options: '--write "**/*.{js,ts,tsx,json,md,css}"'  # Define formats Prettier should apply
 
   Superlinter:
-     runs-on: ubuntu-latest
+    runs-on: ubuntu-latest
+    needs: Prettier  # Ensure Prettier runs first
   
-     steps:
-       - name: Checkout code
-         uses: actions/checkout@v2
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
   
-       - name: Super-Linter
-         uses: super-linter/super-linter@v5.7.2
-         env:
-            VALIDATE: "javascript"  # Only validate JavaScript
-            VALIDATE_ALL_CODEBASE: false
-            VALIDATE_MARKDOWN: false
-            DEFAULT_BRANCH: main 
-            GITHUB_TOKEN: ${{ secrets.TOKEN }} 
-            RUN_LOCAL: false
-            OUTPUT_DETAILS: 'minimal'  # Use minimal output
-            EXCLUDE_PATHS: "/github/workspace/client/node_modules/*,/github/workspace/server/node_modules/*"
+      - name: Run Super-Linter
+        uses: github/super-linter@latest
```

Not sure why the change to this version as it has bugs and is not as stable
```suggestion
        uses: github/super-linter@latest
```